### ISSUE TITLE
CodeChef DSA Learning Series Answers

### ISSUE THREAD

**Issue by puranjay123** [thepranaygupta/Data-Structures-and-Algorithms#215]
hey , Can I work on CodeChef Problem? I will add answers of the DSA learning Series in C++ .

**Comment by thepranaygupta** [comment]
sure @puranjay123 go for it 🚀

**Comment by thepranaygupta** [comment]
> 
> 
> @thepranaygupta is it Hacktoberfest Verified?

Yes it is

**Comment by 8-bit-souvik** [comment]
thanks @puranjay123 for contributing  🎉

**Comment by puranjay123** [comment]
@8-bit-souvik  Why are you closing this issue it is still incomplete and many questions have been left to be uploaded.  I will Complete this folder then you can close this issue or I can simply make PR without the issue to complete this folder.

**Comment by 8-bit-souvik** [comment]
@puranjay123  I reviewed your PR #346, thought it's your last PR

**Comment by puranjay123** [comment]
@8-bit-souvik  No, the folder is not complete once the folder is complete you can close it.

**Comment by puranjay123** [comment]
@8-bit-souvik a friend of mine also uploaded some answers on this folder we have discussed completing this folder together if you don't mind please check his PR's and merge it.

**Comment by thepranaygupta** [comment]
> 
> 
> @8-bit-souvik No, the folder is not complete once the folder is complete you can close it.

sorry for the inconvenience
